tests/integration/conftest.py

```diff
@@ -105,6 +136,8 @@ def patch_completion(monkeypatch, request):
 
 
 def set_up():
+    global cur_id
+    cur_id = 0
```

Comment:
Why set a global `cur_id`

Reply:
This is stolen from an old pr from @rbren 

The idea is that the logs are in sequential order - 001.log, 002.log, etc.

By maintaining an incremental global id, we can directly look up the corresponding response log file.

Reply:
Get it